src/System.Private.CoreLib/src/System/Reflection/MdImport.cs

```diff
@@ -179,18 +180,15 @@ internal struct ConstArray
         }
 
         // Keep the definition in sync with vm\ManagedMdImport.hpp
-        internal int m_length;
-        internal IntPtr m_constArray;
+        internal readonly int m_length;
```

Comment:
Nit: You can move the fields to be first to follow the coding conventions.

Reply:
Done